cli/ops/workers.rs

```diff
@@ -15,6 +16,36 @@ use futures::Sink;
 use futures::Stream;
 use std;
 use std::convert::From;
+use std::sync::atomic::Ordering;
+
+pub fn init(i: &mut Isolate, s: &ThreadSafeState) {
+  i.register_op(
+    "create_worker",
+    s.core_op(json_op(s.stateful_op(op_create_worker))),
+  );
+  i.register_op(
+    "host_get_worker_closed",
+    s.core_op(json_op(s.stateful_op(op_host_get_worker_closed))),
+  );
+  i.register_op(
+    "host_post_message",
+    s.core_op(json_op(s.stateful_op(op_host_post_message))),
+  );
+  i.register_op(
+    "host_get_message",
+    s.core_op(json_op(s.stateful_op(op_host_get_message))),
+  );
+  // TODO: make sure these two ops are only accessible to appropriate Worker
+  i.register_op(
+    "worker_post_message",
+    s.core_op(json_op(s.stateful_op(op_worker_post_message))),
+  );
+  i.register_op(
+    "worker_get_message",
+    s.core_op(json_op(s.stateful_op(op_worker_get_message))),
+  );
+  i.register_op("metrics", s.core_op(json_op(s.stateful_op(op_metrics))));
+}
```

Comment:
Not really a big fan of moving `metrics` to `workers`, but whatever. Regarding the TODO comment I think we could split this file into `worker_host.rs` and `worker.rs`, but again let's do that in follow up PR.

Reply:
> Not really a big fan of moving metrics to workers

Yeah me neither. I don't like having all these op modules with only one op... It seems like resources, performance, metrics, workers could all be combined into something like "internal"... But let's do that later.